[Asset Management] Osquery autocomplete

lykkin opened the pull request:
## Summary

Enables osquery autocomplete. Adds schema files auto generated using `PYTHONPATH=./build/python_path ./tools/codegen/genwebsitejson.py --specs=./specs` in the osquery repo after building. Includes schema files for the latest two versions of osquery.
![autocomplete](https://user-images.githubusercontent.com/1342804/110559405-012efa00-80f9-11eb-9a27-5fb0e530f860.gif)

elasticmachine commented:
Pinging @elastic/security-asset-management (Team:Asset Management)

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112351/)
* Commit: 968eec64e305f539fb984867bd4c0db9ebd3b375
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Setup Build Environment and Dependencies](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112351/execution/node/161/log)
* [Check Types](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112351/execution/node/450/log)
* [Setup Build Environment and Dependencies](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112351/execution/node/163/log)

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @lykkin

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112351/","number":"112351","commit":"968eec64e305f539fb984867bd4c0db9ebd3b375"}]}
PIPELINE-->

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112365/)
* Commit: 68677596476208df078a401e730089da30d205b1
* Storybooks not built
* [Pipeline Steps](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112365/flowGraphTable) (look for red circles / failed steps)
* [Interpreting CI Failures](https://www.elastic.co/guide/en/kibana/current/interpreting-ci-failures.html)

### Failed CI Steps
* [Build X-Pack Plugins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112365/execution/node/182/log)
* [Report failed tests, if necessary](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112365/execution/node/194/log)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ‼️ ERROR: metrics for 68677596476208df078a401e730089da30d205b1 were not reported

### History
* :broken_heart: [Build #112351](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112351/) failed 968eec64e305f539fb984867bd4c0db9ebd3b375

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @lykkin

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112351/","number":"112351","commit":"968eec64e305f539fb984867bd4c0db9ebd3b375"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112365/","number":"112365","commit":"68677596476208df078a401e730089da30d205b1"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112371/)
* Commit: 2872fb252352e53fdc0d4198ccd31980aabde476
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-94255/2872fb252352e53fdc0d4198ccd31980aabde476)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/ec28b780dba5b593a2e69e3cd6f5d93fc732a225) | [after](https://github.com/elastic/kibana/commit/2872fb252352e53fdc0d4198ccd31980aabde476) | diff |
| --- | --- | --- | --- |
| `osquery` | 131 | 136 | +5 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/ec28b780dba5b593a2e69e3cd6f5d93fc732a225) | [after](https://github.com/elastic/kibana/commit/2872fb252352e53fdc0d4198ccd31980aabde476) | diff |
| --- | --- | --- | --- |
| `osquery` | 318.2KB | 1.1MB | :warning: +844.3KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/ec28b780dba5b593a2e69e3cd6f5d93fc732a225) | [after](https://github.com/elastic/kibana/commit/2872fb252352e53fdc0d4198ccd31980aabde476) | diff |
| --- | --- | --- | --- |
| `osquery` | 12.6KB | 12.7KB | +66.0B |

### History
* :broken_heart: [Build #112365](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112365/) failed 68677596476208df078a401e730089da30d205b1
* :broken_heart: [Build #112351](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112351/) failed 968eec64e305f539fb984867bd4c0db9ebd3b375

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @lykkin

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112351/","number":"112351","commit":"968eec64e305f539fb984867bd4c0db9ebd3b375"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112365/","number":"112365","commit":"68677596476208df078a401e730089da30d205b1"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112371/","number":"112371","commit":"2872fb252352e53fdc0d4198ccd31980aabde476"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112375/)
* Commit: a5973a60cd945f070c58c3754572aae8fc362270
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-94255/a5973a60cd945f070c58c3754572aae8fc362270)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/ec28b780dba5b593a2e69e3cd6f5d93fc732a225) | [after](https://github.com/elastic/kibana/commit/a5973a60cd945f070c58c3754572aae8fc362270) | diff |
| --- | --- | --- | --- |
| `osquery` | 131 | 136 | +5 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/ec28b780dba5b593a2e69e3cd6f5d93fc732a225) | [after](https://github.com/elastic/kibana/commit/a5973a60cd945f070c58c3754572aae8fc362270) | diff |
| --- | --- | --- | --- |
| `osquery` | 318.2KB | 1.1MB | :warning: +844.3KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/ec28b780dba5b593a2e69e3cd6f5d93fc732a225) | [after](https://github.com/elastic/kibana/commit/a5973a60cd945f070c58c3754572aae8fc362270) | diff |
| --- | --- | --- | --- |
| `osquery` | 12.6KB | 12.7KB | +66.0B |

### History
* :green_heart: [Build #112371](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112371/) succeeded 2872fb252352e53fdc0d4198ccd31980aabde476
* :broken_heart: [Build #112365](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112365/) failed 68677596476208df078a401e730089da30d205b1
* :broken_heart: [Build #112351](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112351/) failed 968eec64e305f539fb984867bd4c0db9ebd3b375

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @lykkin

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112351/","number":"112351","commit":"968eec64e305f539fb984867bd4c0db9ebd3b375"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112365/","number":"112365","commit":"68677596476208df078a401e730089da30d205b1"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112371/","number":"112371","commit":"2872fb252352e53fdc0d4198ccd31980aabde476"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112375/","number":"112375","commit":"a5973a60cd945f070c58c3754572aae8fc362270"}]}
PIPELINE-->

patrykkopycinski commented on x-pack/plugins/osquery/tsconfig.json:
```diff
@@ -10,7 +10,7 @@
     // add all the folders contains files to be compiled
     "common/**/*",
-    "public/**/*",
+    "public/**/*.{json,tsx,ts}",
```
can we instead exclude the files that are causing the issues?

lykkin replied:
for some reason the glob wasn't picking up the schema json file until i explicitly added the `.json` extension. the other way i got this working was to add another entry `public/**/*.json`, which would be more permissive. WDYT?

patrykkopycinski replied:
I think we can go with Canvas team approach like
https://github.com/lykkin/kibana/blob/osquery-autocomplete/x-pack/plugins/canvas/canvas_plugin_src/functions/server/demodata/get_demo_rows.ts#L9
and
https://github.com/lykkin/kibana/blob/osquery-autocomplete/x-pack/plugins/canvas/tsconfig.json#L27

patrykkopycinski commented:
Tried to test it locally, but got an error 
![image](https://user-images.githubusercontent.com/5188868/110628774-b7e7b480-81a3-11eb-8e91-f29081d137b0.png)

![Uploading image.png…]()

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112432/)
* Commit: a5973a60cd945f070c58c3754572aae8fc362270
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-94255/a5973a60cd945f070c58c3754572aae8fc362270)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/ec28b780dba5b593a2e69e3cd6f5d93fc732a225) | [after](https://github.com/elastic/kibana/commit/a5973a60cd945f070c58c3754572aae8fc362270) | diff |
| --- | --- | --- | --- |
| `osquery` | 131 | 136 | +5 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/ec28b780dba5b593a2e69e3cd6f5d93fc732a225) | [after](https://github.com/elastic/kibana/commit/a5973a60cd945f070c58c3754572aae8fc362270) | diff |
| --- | --- | --- | --- |
| `osquery` | 318.2KB | 1.1MB | :warning: +844.3KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/ec28b780dba5b593a2e69e3cd6f5d93fc732a225) | [after](https://github.com/elastic/kibana/commit/a5973a60cd945f070c58c3754572aae8fc362270) | diff |
| --- | --- | --- | --- |
| `osquery` | 12.6KB | 12.7KB | +66.0B |

### History
* :green_heart: [Build #112375](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112375/) succeeded a5973a60cd945f070c58c3754572aae8fc362270
* :green_heart: [Build #112371](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112371/) succeeded 2872fb252352e53fdc0d4198ccd31980aabde476
* :broken_heart: [Build #112365](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112365/) failed 68677596476208df078a401e730089da30d205b1
* :broken_heart: [Build #112351](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112351/) failed 968eec64e305f539fb984867bd4c0db9ebd3b375

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @lykkin

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112351/","number":"112351","commit":"968eec64e305f539fb984867bd4c0db9ebd3b375"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112365/","number":"112365","commit":"68677596476208df078a401e730089da30d205b1"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112371/","number":"112371","commit":"2872fb252352e53fdc0d4198ccd31980aabde476"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112375/","number":"112375","commit":"a5973a60cd945f070c58c3754572aae8fc362270"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112432/","number":"112432","commit":"a5973a60cd945f070c58c3754572aae8fc362270"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112610/)
* Commit: ca3975625988c11bca30fcc9198223329058217a
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-94255/ca3975625988c11bca30fcc9198223329058217a)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/ec28b780dba5b593a2e69e3cd6f5d93fc732a225) | [after](https://github.com/elastic/kibana/commit/ca3975625988c11bca30fcc9198223329058217a) | diff |
| --- | --- | --- | --- |
| `osquery` | 131 | 137 | +6 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/ec28b780dba5b593a2e69e3cd6f5d93fc732a225) | [after](https://github.com/elastic/kibana/commit/ca3975625988c11bca30fcc9198223329058217a) | diff |
| --- | --- | --- | --- |
| `osquery` | 318.2KB | 1.1MB | :warning: +846.5KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/ec28b780dba5b593a2e69e3cd6f5d93fc732a225) | [after](https://github.com/elastic/kibana/commit/ca3975625988c11bca30fcc9198223329058217a) | diff |
| --- | --- | --- | --- |
| `osquery` | 12.6KB | 12.7KB | +66.0B |

### History
* :green_heart: [Build #112432](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112432/) succeeded a5973a60cd945f070c58c3754572aae8fc362270
* :green_heart: [Build #112375](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112375/) succeeded a5973a60cd945f070c58c3754572aae8fc362270
* :green_heart: [Build #112371](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112371/) succeeded 2872fb252352e53fdc0d4198ccd31980aabde476
* :broken_heart: [Build #112365](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112365/) failed 68677596476208df078a401e730089da30d205b1
* :broken_heart: [Build #112351](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112351/) failed 968eec64e305f539fb984867bd4c0db9ebd3b375

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @lykkin

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112351/","number":"112351","commit":"968eec64e305f539fb984867bd4c0db9ebd3b375"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112365/","number":"112365","commit":"68677596476208df078a401e730089da30d205b1"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112371/","number":"112371","commit":"2872fb252352e53fdc0d4198ccd31980aabde476"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112375/","number":"112375","commit":"a5973a60cd945f070c58c3754572aae8fc362270"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112432/","number":"112432","commit":"a5973a60cd945f070c58c3754572aae8fc362270"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112610/","number":"112610","commit":"ca3975625988c11bca30fcc9198223329058217a"}]}
PIPELINE-->

patrykkopycinski reviewed (approved):
Tested it locally and works great 👏  Thank you @lykkin 
Could you try to minify the schema json before merge?

lykkin commented:
@elasticmachine merge upstream

lykkin commented:
>Could you try to minify the schema json before merge?

i added a script to pare things down to a set of fields that we want. as a result i removed most of the file, aside from the top level table `name` field, since that's all we are currently using. it should be easy enough to add data back in as we want to reference different types (e.g. populate column level auto complete, or docs for different versions/platforms in the future)

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112914/)
* Commit: 5545e34b92813e62454edce81c78fcef6dfdf685
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Check TypeScript Projects](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112914/execution/node/439/log)

### History
* :green_heart: [Build #112610](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112610/) succeeded ca3975625988c11bca30fcc9198223329058217a
* :green_heart: [Build #112432](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112432/) succeeded a5973a60cd945f070c58c3754572aae8fc362270
* :green_heart: [Build #112375](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112375/) succeeded a5973a60cd945f070c58c3754572aae8fc362270
* :green_heart: [Build #112371](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112371/) succeeded 2872fb252352e53fdc0d4198ccd31980aabde476
* :broken_heart: [Build #112365](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112365/) failed 68677596476208df078a401e730089da30d205b1

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @lykkin

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112365/","number":"112365","commit":"68677596476208df078a401e730089da30d205b1"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112371/","number":"112371","commit":"2872fb252352e53fdc0d4198ccd31980aabde476"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112375/","number":"112375","commit":"a5973a60cd945f070c58c3754572aae8fc362270"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112432/","number":"112432","commit":"a5973a60cd945f070c58c3754572aae8fc362270"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112610/","number":"112610","commit":"ca3975625988c11bca30fcc9198223329058217a"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112914/","number":"112914","commit":"5545e34b92813e62454edce81c78fcef6dfdf685"}]}
PIPELINE-->

lykkin commented:
@elasticmachine merge upstream

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112990/)
* Commit: 7e2a3b5a16ae6f70538050583825b70aeaf5e928
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Check Types](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112990/execution/node/449/log)

### History
* :broken_heart: [Build #112914](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112914/) failed 5545e34b92813e62454edce81c78fcef6dfdf685
* :green_heart: [Build #112610](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112610/) succeeded ca3975625988c11bca30fcc9198223329058217a
* :green_heart: [Build #112432](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112432/) succeeded a5973a60cd945f070c58c3754572aae8fc362270
* :green_heart: [Build #112375](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112375/) succeeded a5973a60cd945f070c58c3754572aae8fc362270
* :green_heart: [Build #112371](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112371/) succeeded 2872fb252352e53fdc0d4198ccd31980aabde476

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @lykkin

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112371/","number":"112371","commit":"2872fb252352e53fdc0d4198ccd31980aabde476"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112375/","number":"112375","commit":"a5973a60cd945f070c58c3754572aae8fc362270"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112432/","number":"112432","commit":"a5973a60cd945f070c58c3754572aae8fc362270"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112610/","number":"112610","commit":"ca3975625988c11bca30fcc9198223329058217a"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112914/","number":"112914","commit":"5545e34b92813e62454edce81c78fcef6dfdf685"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112990/","number":"112990","commit":"7e2a3b5a16ae6f70538050583825b70aeaf5e928"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112993/)
* Commit: 43b5d40f40543ef39f50a0f11051818ade1bb168
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-94255/43b5d40f40543ef39f50a0f11051818ade1bb168)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/0c3514c0f7e6a12ee4a1e09c65a478a3a9a32016) | [after](https://github.com/elastic/kibana/commit/43b5d40f40543ef39f50a0f11051818ade1bb168) | diff |
| --- | --- | --- | --- |
| `osquery` | 131 | 137 | +6 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/0c3514c0f7e6a12ee4a1e09c65a478a3a9a32016) | [after](https://github.com/elastic/kibana/commit/43b5d40f40543ef39f50a0f11051818ade1bb168) | diff |
| --- | --- | --- | --- |
| `osquery` | 318.2KB | 802.4KB | :warning: +484.2KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/0c3514c0f7e6a12ee4a1e09c65a478a3a9a32016) | [after](https://github.com/elastic/kibana/commit/43b5d40f40543ef39f50a0f11051818ade1bb168) | diff |
| --- | --- | --- | --- |
| `osquery` | 12.6KB | 12.7KB | +66.0B |

### History
* :broken_heart: [Build #112990](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112990/) failed 7e2a3b5a16ae6f70538050583825b70aeaf5e928
* :broken_heart: [Build #112914](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112914/) failed 5545e34b92813e62454edce81c78fcef6dfdf685
* :green_heart: [Build #112610](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112610/) succeeded ca3975625988c11bca30fcc9198223329058217a
* :green_heart: [Build #112432](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112432/) succeeded a5973a60cd945f070c58c3754572aae8fc362270
* :green_heart: [Build #112375](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112375/) succeeded a5973a60cd945f070c58c3754572aae8fc362270

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @lykkin

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112375/","number":"112375","commit":"a5973a60cd945f070c58c3754572aae8fc362270"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112432/","number":"112432","commit":"a5973a60cd945f070c58c3754572aae8fc362270"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112610/","number":"112610","commit":"ca3975625988c11bca30fcc9198223329058217a"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112914/","number":"112914","commit":"5545e34b92813e62454edce81c78fcef6dfdf685"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112990/","number":"112990","commit":"7e2a3b5a16ae6f70538050583825b70aeaf5e928"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/112993/","number":"112993","commit":"43b5d40f40543ef39f50a0f11051818ade1bb168"}]}
PIPELINE-->

kibanamachine commented:
## 💚 Backport successful

✅ [7.x](https://github.com/elastic/kibana/pull/94584) / https://github.com/elastic/kibana/pull/94584

Successful backport PRs will be merged automatically after passing CI.
